wordpress-mobile/WordPress-FluxC-Android
# Woo: Batch query to prevent SQLiteException crash

Fixes #1477 by batching the request to fetch `WCOrderSummaryModel` records by `remoteOrderId` into chunks of 200 records. 

### To Test
- Run the example app -> Woo -> Orders -> Order List, and swipe through the results to load as many pages of orders as are available in test site and verify no issues. 
- Run the `OrderSqlUtilsTest.testGetOrderSummariesByRemoteIds()` unit test
